scripts/analyze_github_actions_usage.py

```
@@ -0,0 +1,390 @@
+        print("\nUsage:")
+        print("  export GITHUB_TOKEN=your_token_here")
+        print("  python scripts/analyze_github_actions_usage.py")
+        print("\nOr use gh CLI:")
+        print("  gh auth token | python scripts/analyze_github_actions_usage.py --use-stdin-token")
+        sys.exit(1)
+
+    # Get repository info from git remote
+    try:
+        import subprocess
+        remote_url = subprocess.check_output(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)